ciscoise_connector.py

```diff
@@ -648,24 +670,34 @@ def _clear_policy(self, param):
         action_result.add_data(ret_data)
         return action_result.set_status(phantom.APP_SUCCESS, "Policy cleared")
 
-    def _test_connectivity(self, param):
-
-        rest_endpoint = '{0}/{1}'.format(self._base_url, ACTIVE_COUNT_REST_ENDPOINT)
-
-        config = self.get_config()
-
-        self.save_progress(phantom.APP_PROG_CONNECTING_TO_ELLIPSES, config[phantom.APP_JSON_DEVICE])
-        verify = config[phantom.APP_JSON_VERIFY]
-
+    def _test_connectivity_to_device(self, device_domain, verify=True):
         try:
+            rest_endpoint = 'https://{0}{1}'.format(device_domain, ACTIVE_LIST_REST)
+            self.save_progress(phantom.APP_PROG_CONNECTING_TO_ELLIPSES, device_domain)
             resp = requests.get(rest_endpoint, auth=self._auth, verify=verify)
         except Exception as e:
             return self.set_status(phantom.APP_ERROR, CISCOISE_ERR_TEST_CONNECTIVITY_FAILED, e)
 
-        if resp.status_code != 200:
+        if resp.status_code == 200:
+            return self.set_status_save_progress(phantom.APP_SUCCESS, CISCOISE_SUCC_TEST_CONNECTIVITY_PASSED)
```

Comment:
Can we have separate success messages for both primary and secondary devices?

Reply:
This is a helper method for test_connectivity. This method is called from test_connectivity once if only one node is present and twice if two nodes are present. And message is displayed for each node according to return value of this method. Actual test connectivity is below at line 686